src/Voron/Data/Tables/Table.cs

```diff
@@ -1189,6 +1189,9 @@ public IEnumerable<SeekResult> SeekForwardFrom(TableSchema.AbstractTreeIndexDef
         public IEnumerable<SeekResult> SeekForwardFromPrefix(TableSchema.AbstractTreeIndexDef index, Slice start, Slice prefix, long skip)
         {
             var tree = GetTree(index);
+            if (tree == null)
```

Comment:
When can this happen?

Reply:
`tree == null` can happen for example, if we call `GetDocumentsByBucketFrom` when we don't have any documents on this shard yet.

`GetDocumentsByBucketFrom` tries to do 
```
foreach (var result in table.SeekForwardFromPrefix(DocsSchema.DynamicKeyIndexes[DocsBucketAndEtagSlice], keySlice, prefix, 0))
```

but `DocsBucketAndEtagSlice` index-tree doesn't exist yet, it is created in `DocsSchema.Create`, upon first document-put :   
https://github.com/ravendb/ravendb/blob/feature/sharding/src/Raven.Server/Documents/DocumentsStorage.cs#L2483